src/Network/Session.php

```diff
@@ -108,7 +108,9 @@ public static function create($sessionConfig = [])
             $sessionConfig['ini']['session.name'] = $sessionConfig['cookie'];
         }
 
-        if (!empty($sessionConfig['handler'])) {
+        // In PHP7.1.0+ session.save_handler can't be set to user by the user.
+        // https://github.com/php/php-src/blob/master/ext/session/session.c#L559
+        if (!empty($sessionConfig['handler']) && version_compare(PHP_VERSION, '7.1.0', '<=')) {
```

Comment:
No in 7.1 this simply did nothing from what I could figure out of the php source code.